apps/web/src/hooks/affine/use-current-user.ts

```diff
@@ -16,16 +29,32 @@ export function useCurrentUser(): CheckedUser {
   const session = useSession();
   // If you are seeing this error, it means that you are not logged in.
   //  This should be prohibited in the development environment, please re-write your component logic.
-  assertEquals(
-    session.status,
-    'authenticated',
-    'session.status should be authenticated'
-  );
-  const user = session.data.user;
+  // assertEquals(
+  //   session.status,
+  //   'authenticated',
+  //   'session.status should be authenticated'
+  // );
```

Comment:
UNCOMMENT this. This assertion is meant to make sure you are using this hook in the right component